core/src/main/scala/cromwell/core/labels/Labels.scala

```diff
@@ -20,7 +22,19 @@ case class Labels(value: Vector[Label]) {
 object Labels {
   def apply(values: (String, String)*): Labels = {
     val kvps: Seq[(String, String)] = values.toSeq
-    Labels((kvps map { case (k, v) => Label.safeLabel(k, v) }).to[Vector])
+    Labels((kvps map { case (k, v) => Label(k, v) }).to[Vector])
+  }
+
+  //This method should only ever be used by the JES backend as this manipulation
+  //to safe google label names is required to guard against JES job submission failure.
+  def googleLabels(values: (String, String)*): Labels = {
```

Comment:
JES specific stuff should probably live in the JES backend package (probably via an enhancer class) rather than living here in the core package.